types/combine-source-map/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for combine-source-map 0.8
+// Project: https://github.com/thlorenz/combine-source-map
+// Definitions by: TeamworkGuy2 <https://github.com/TeamworkGuy2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/** Add source maps of multiple files, offset them and then combine them into one source map.
+ * (documentation based on project's README file)
+ */
+declare namespace CombineSourceMap {
```

Comment:
Instead of declaring a namespace, move everything here to the top level and directly export.